lib/cc/analyzer/engines_config_builder.rb

```diff
@@ -11,11 +11,11 @@ class EnginesConfigBuilder
         :container_label,
       )
 
-      def initialize(registry:, config:, container_label:, source_dir:, include_paths:)
+      def initialize(registry:, config:, container_label:, source_dir:, include_paths_builder:)
```

Comment:
Right, I'm 100% unaware of builder and how it works so I want to make sure whatever change I make here is going to be compatible / what we want to do while also keeping the startup time of each engine down in the CLI.